### PR TITLE
update machine instance configs

### PR DESCRIPTION
**Pull request by amitsagtani97**
### Description

Migrate to m6i and gp3 instance types

### CODE REVIEW

**Review by mohit-sheth — APPROVED** [review]
LGTM

File: dags/openshift_nightlies/config/install/aws/acs.json [hunk 1]
```diff
@@ -7,26 +7,26 @@
...
-    "openshift_infra_node_volume_type": "gp2",
+    "openshift_infra_node_volume_type": "gp3",
     "openshift_infra_node_volume_iops": 0,
     "openshift_workload_node_volume_size": 500,
-    "openshift_workload_node_volume_type": "gp2",
+    "openshift_workload_node_volume_type": "gp3",
     "openshift_workload_node_volume_iops": 0,
     "openshift_prometheus_retention_period": "15d",
-    "openshift_prometheus_storage_class": "gp2",
+    "openshift_prometheus_storage_class": "gp3-csi",
```
Comment (rsevilla87) [inline dags/openshift_nightlies/config/install/aws/acs.json:27]: Can we use the original gp2 storageclass to ensure this doesn't break 4.9 installations?

Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/aws/acs.json:27]: @amitsagtani97

Comment (amitsagtani97) [reply dags/openshift_nightlies/config/install/aws/acs.json]: Updated :+1:

**Review by morenod — APPROVED** [review]
LGTM

**Pull request merged**